src/util/http_utils.js

```diff
@@ -23,6 +23,7 @@ const cloud_utils = require('./cloud_utils');
 
 const UNSIGNED_PAYLOAD = 'UNSIGNED-PAYLOAD';
 const STREAMING_PAYLOAD = 'STREAMING-AWS4-HMAC-SHA256-PAYLOAD';
+const FORM_URL_ENCODED = 'application/x-www-form-urlencoded';
```

Comment:
This eventually should be a list of constants for the content types we commonly use in our code base, see suggestion.
Also notice there's another occurance of 'x-www-form-urlencoded' in this file in function `parse_request_body()`, better replace it with this constant too.

```suggestion
const CONTENT_TYPE_TEXT_PLAIN = 'text/plain';
const CONTENT_TYPE_APP_OCTET_STREAM = 'application/octet-stream';
const CONTENT_TYPE_APP_JSON = 'application/json';
const CONTENT_TYPE_APP_XML = 'application/xml';
const CONTENT_TYPE_APP_FORM_URLENCODED = 'application/x-www-form-urlencoded';
```

Reply:
I added the constants, I will reuse the new `CONTENT_TYPE_APP_FORM_URLENCODED` in other places in the code (that are not IAM) in the PR where I'm going to refactor S3 and STS.
In this PR I'm trying to keep the changes only to IAM.